packages/grafana-prometheus/src/components/monaco-query-field/monaco-completion-provider/situation.ts

```diff
@@ -367,6 +377,61 @@ function resolveLabelMatcher(node: SyntaxNode, text: string, pos: number): Situa
   };
 }
 
+function resolveQuotedLabelMatcher(node: SyntaxNode, text: string): Situation | null {
```

Comment:
I'll come up with a better solution instead of just duplicating the code. 
Duplicated code is coming from `resolveLabelMatcher`.